### PR TITLE
Vk ext conservative rasterization

### PR DESCRIPTION
**Pull request by ethan-conffx**
Adds a sample application that tests VK_EXT_conservative_rasterization

### CODE REVIEW

File: application_sandbox/conservative_rasterization/main.cpp [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+    conservative_rasterization_pipeline_->SetTopology(
+        VK_PRIMITIVE_TOPOLOGY_TRIANGLE_LIST);
+    conservative_rasterization_pipeline_->SetInputStreams(&cube_);
+    conservative_rasterization_pipeline_->SetViewport(viewport());
+    conservative_rasterization_pipeline_->SetScissor(scissor());
+    conservative_rasterization_pipeline_->SetSamples(num_samples());
+    conservative_rasterization_pipeline_->AddAttachment();
+
+	// Setup conservative rasterization
+    PFN_vkGetPhysicalDeviceProperties2KHR vkGetPhysicalDeviceProperties2KHR =
```
Comment (AWoloszyn) [inline application_sandbox/conservative_rasterization/main.cpp]: You can add vkGetPhysicalDeviceProperties2KHR to  `vulkan_wrapper/function_table.h`, which would simplify this.
(It wont get resolved unless you use it, so it should be safe)

Comment (ethan-conffx) [reply application_sandbox/conservative_rasterization/main.cpp]: Thanks for letting me know. I will move it to `vulkan_wrapper/function_table.h`

**Comment by ethan-conffx** [comment]
@googlebot I signed it!

**Comment by ethan-conffx** [comment]
@googlebot I signed it!

**Comment by AWoloszyn** [comment]
This should be squashed and rebased into a single commit. (As well to remove the merge conflict) other than that looks good.

**Pull request merged**